src/UInt.dfy

```diff
@@ -0,0 +1,335 @@
+// RUN: %dafny /compile:0 "%s"
+
+/*******************************************************************************
+*  Copyright by the contributors to the Dafny Project
+*  SPDX-License-Identifier: MIT 
+*******************************************************************************/
+
+module Helpers.UInt {
+
+  newtype uint8 = x | 0 <= x < 0x100
```

Comment:
Would it be possible to move all these definitions to BoundedInts.dfy ? I did not see that this file existed, and that would be handy as well if you could incorporate the definitions of this upcoming PR:

https://github.com/dafny-lang/libraries/pull/51/files#diff-437e3168477b72c1dc1b4f5e935632c1f00659c1497c7f8d673ec2de1a1efa69R36